### PR TITLE
[main] fix tsdoc error for param and type tag

### PR DESCRIPTION
**Pull request by siyuniu-ms**

### CODE REVIEW

File: AISKU/src/AISku.ts [hunk 1]
```diff
@@ -626,33 +626,29 @@ export class AppInsightsSku implements IApplicationInsights {
 
     /**
      * Log a user action or other occurrence.
-     * @param event
-     * @param [customProperties]
+     * @param customProperties - properties that would be included as part of the event
```
Comment (Karlie-777) [inline AISKU/src/AISku.ts:629]: maybe we should add back param event

File: shared/AppInsightsCore/consolelog.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
+
+> @microsoft/applicationinsights-core-js@3.3.4 dtsgen
```
Comment (Karlie-777) [inline shared/AppInsightsCore/consolelog.md:2]: Is it generated by rush?

File: shared/AppInsightsCommon/src/Interfaces/ITraceTelemetry.ts [hunk 3]
```diff
@@ -7,19 +7,16 @@ import { IPartC } from "./IPartC";
 export interface ITraceTelemetry extends IPartC {
     /**
      * @description A message string
-     * @type {string}
```
Comment (Karlie-777) [inline shared/AppInsightsCommon/src/Interfaces/ITraceTelemetry.ts:10]: maybe we should keep type as well?

File: shared/AppInsightsCore/consolelog.md (hunk 2 above)
Comment (siyuniu-ms) [reply shared/AppInsightsCore/consolelog.md]: deleted

**Review by Karlie-777 — APPROVED** [review]

**Pull request merged**